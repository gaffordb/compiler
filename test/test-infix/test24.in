let useless : int->unit = fun (x:int) : unit |-> ()
in
useless 1
