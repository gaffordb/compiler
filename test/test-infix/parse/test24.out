(let useless = (fun x -> ()) in
(useless 1))
